packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -670,6 +671,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             }
 
             const clientTime = networkTime ? overallTime - networkTime : undefined;
+            const isAfd = msEdge != null ? true : false;
```

Comment:
Will it be null or undefined if not present? I think undefined, not sure though. Please check.

Reply:
its undefined. Corrected thanks!